src/mpid/ch4/src/ch4_init.c

```diff
@@ -79,6 +79,18 @@
         handoff
         trylock
 
+    - name        : MPIR_CVAR_CH4_EAGER_MAX_MSG_SIZE
```

Comment:
Is there any reason outside of debugging to set this value? I'm a little concerned about how others may view this value if they go digging in the CVARs.

Reply:
Probably not, though people sometimes want to tweak the eager/rendezvous threshold

Reply:
Alright. Looking closer at ch3 this seems fine.